app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
+
+import { calculateCreatedAgo, calculateDateToWeekday } from "./utils";
+
+const Note = ({ note }) => (
+  <div className="neeto-ui-shadow-xs mb-4 w-full gap-3 space-y-2 border py-2 px-4">
+    <div className="flex justify-between ">
+      <Typography style="h4">{note.title}</Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <div className="mx-2">
+          <li>Edit</li>
+          <li>Delete</li>
+        </div>
+      </Dropdown>
+    </div>
+    <Typography className="neeto-ui-text-gray-600" style="body2">
+      {note.description}
+    </Typography>
+    <hr />
+    <div className="flex justify-between">
+      <Tag
+        className="neeto-ui-rounded-sm neeto-ui-bg-gray-100 box-border h-6 px-2 py-px"
+        label="Getting Started"
+        style="body3"
+      />
+      <div className="flex items-center justify-between">
+        <Clock color="#68737d" size="20" />
+        <Tooltip
+          content={calculateDateToWeekday(note.created_at)}
+          followCursor="horizontal"
+          position="bottom"
+        >
+          <Typography
+            className="neeto-ui-text-gray-600 ml-1 mr-2"
+            style="body3"
+          >
+            Created {calculateCreatedAgo(note.created_at)}
+          </Typography>
+        </Tooltip>
+        <Avatar
+          user={{
+            imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
+          }}
+          onClick={function noRefCheck() {}}
```

Comment:
Go through the Slack thread very carefully and fix the issue here.

Reply:
I have removed the onClick function as it there was no action to be performed when the avatar is clicked. I will be careful to use the arrow function next time in accordance with the react best practices listed in the slack thread.

> @sunnyshahabuddin Also go through https://handbook.neetokb.com/articles/git-and-github-essentials very carefully and see if you need to make any changes in this PR>
I have also changed pull request title as per "git and github essentials". I will make sure to cross check following points before creating a pull request next time:

1. Title should always begin with a capital letter and not with a number.
2. Link the pull request with the associated issue.
3. Write the first comment mentioning you, and to ask for a review.

Reply:
>Please let me know for any other changes, or points that I missed.

Sounds good. You haven't missed any points now. 👍

Reply:
Thank you Yedhin.